api/views.py

```diff
@@ -138,3 +143,17 @@ def _formatted_bad_request(format_):
     else:
         # json
         return JsonResponse({}, status=400)
+
+@api_view([GET])
+def enrichment_view(request):
```

Comment:
For best practice, you should use 2 serializers for this view: `IoCSerializer` and `EnrichmentSerializer`.
- The `EnrichmentSerializer` should have 3 fields: 
    - `query` as a CharField with required, write_only attributes set to True.
    - `found` as a `BooleanField(read_only=True, default=False`). 
    - `ioc = IoCSerializer(read_only=True, default=null)`.
- The `regex_validate` function should be used as the the `validate_query` method in `EnrichmentSerializer`.
- The IoC.objects.get check should be inside the `validate` method of `EnrichmentSerializer`. if was found, set appropriately the ioc and found fields.